### PR TITLE
fix get target_version failed

### PR DESCRIPTION
**Pull request by Ximinhan**
fix failure in [job](https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/scheduled-builds/job/sync-ci-images/74732/consoleFull)

### CODE REVIEW

File: doozer/doozerlib/cli/images_streams.py [hunk 1]
```diff
@@ -738,7 +738,7 @@ def search_issues(query):
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
             'versions': [{'name': release_version}],  # Affects Version/s
-            'customfield_12323140': [{'name': Model(runtime.gitdata.load_data(key='bug').data).jira_config.target_release[-1]}],  # customfield_12323140 is Target Version in jira
+            'customfield_12323140': [{'name': runtime.gitdata.load_data(key='bug').data.get('target_release')[-1]}],  # customfield_12323140 is Target Version in jira
```
Comment (vfreex) [inline doozer/doozerlib/cli/images_streams.py:741]: Properties under `jira_config` need to be merged into the parent object. See https://github.com/openshift/elliott/blob/18aef26c32b01bebd2983871467c75927ab43b9a/elliottlib/bzutil.py#L583

**Review by vfreex — CHANGES_REQUESTED** [review]

File: doozer/doozerlib/cli/images_streams.py (hunk 1 above)
Comment (Ximinhan) [reply doozer/doozerlib/cli/images_streams.py:741]: what do you mean? target_release is not under jira_config

Comment (vfreex) [reply doozer/doozerlib/cli/images_streams.py:741]: Please read the code I linked. Properties under `jira_config` (including `target_release`) are merged into the top-level object of `bug.yml`.

Comment (vfreex) [reply doozer/doozerlib/cli/images_streams.py:741]: That means target_release can be a field of either jira_config or the top level object.

**Comment by Ximinhan** [comment]
already fixed by https://github.com/openshift-eng/art-tools/pull/15

**Pull request closed without merging**